markdown-pages/en/tidb/master/releases/release-7.0.0.md

```diff
@@ -0,0 +1,86 @@
+---
+title: TiDB 7.0.0 Release Notes
+---
+
+# TiDB 7.0.0  Release Notes
+
+Release date: xx, 2023
+
+TiDB version: 7.0.0-DMR
+
+Quick access: Quick start | Installation package
+
+In v7.0.0-DMR, the key new features and improvements are as follows:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature/Enhancement</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">Scalability and Performance<br/></td>
+    <td>Session level SQL plan cache (experimental)</td>
+    <td>Using plan cache can reduce the time for compiling and optimizing SQL queries, which reduces query latency. For each query executed, if a plan doesn't exist, a new one will be stored and re-used next time.</td>
+  </tr>
+  <tr>
+    <td>TiFlash supports storage-computing separation and S3 shared storage (experimental)</td>
+    <td>TiFlash introduces a new architecture:
+      <ol>
+        <li>Disaggregate TiFlash's compute and storage which is a milestone for elastic HTAP resource utilization.</li>
+        <li>Introduces S3 based storage engine which can provide shared storage at lower cost.</li>
+      </ol>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2">Reliability and Availability<br/></td>
+    <td>Resource control enhancement (GA) </td>
+    <td>Users can now use resource groups to allocate and isolate resources for different applications or workloads in one cluster. This supports different resource binding modes (user, session, query) and user-defined priority. Quotas can be "burstable".</td>
```

Comment:
Users now can use resource groups to allocate and isolate resources for different applications or workloads in one cluster. In this version, TiDB added supports for different resource binding modes (user, session, query) and user-defined priority, users can also use command to  do resource calibration(estimation for whole resource amount).